charts/barman/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: barman
 type: application
 description: Chart for Barman PostgreSQL Backup and Recovery Manager
-version: 0.0.13
+version: 0.0.14
```

Because this is a breaking change I would increase the minor version, even if we're still in the 0.x release cycle.

```suggestion
version: 0.1.0
```